File: wear/src/main/java/brendan/wattsapp/WearDisplayActivity.java
package brendan.wattsapp;

import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;

public class WearDisplayActivity extends Activity {

    private TextView mTextView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_display);
        mTextView = (TextView) findViewById(R.id.text);
    }
}

File: wear/src/main/kotlin/brendan/wattsapp/WearDisplayActivity.kt
package brendan.wattsapp

import android.app.Activity
import android.os.Bundle
import android.widget.TextView

class WearDisplayActivity : Activity() {

    private var mTextView: TextView? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_display)
        mTextView = findViewById(R.id.text) as TextView
    }
}
